File: src/main/java/com/github/ojacquemart/ApimobileApplication.java
package com.github.ojacquemart;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableScheduling
public class ApimobileApplication {

	public static void main(String[] args) {
		SpringApplication.run(ApimobileApplication.class, args);
	}
}

File: src/main/kotlin/com/github/ojacquemart/api/restaurant/ApimobileApplication.kt
package com.github.ojacquemart.api.restaurant

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.cache.annotation.EnableCaching
import org.springframework.scheduling.annotation.EnableScheduling

@EnableCaching
@EnableConfigurationProperties
@EnableScheduling
@SpringBootApplication
open class ApiMobileApplication

fun main(args: Array<String>) {
    SpringApplication.run(ApiMobileApplication::class.java, *args)
}
